src/builder/modules/Generator.res

```diff
@@ -122,10 +122,31 @@ let usePage = id => {
   model := Builder.usePage(model.contents, id)
 }
 
-let fillBackgroundColor = (fillStyle: string) => {
-  model := Generator_Builder.fillBackgroundColor(model.contents, fillStyle)
+let fillBackgroundColor = (~color: string="#ffffff", ()) => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, ~color)
+}
+let drawLine = (
+  dest: Builder.rectangle,
+  ~color: string="#000000",
+  ~width: float=1.0,
+  ~pattern: array<int>=[],
+  ~offset: int=0,
```

Comment:
To keep this very simple, could I suggest we remove the `pattern` and `offset` arguments.

The goal of the generator is not to expose canvas drawing APIs, but your suggestion expose some simple line and shape functions is a good idea.